server/network.js

```diff
@@ -356,27 +393,27 @@ network.readNetwork_ = function(networkFile) {
     if (source in nodeId) {
       nodes[nodeId[source]].isTF = true;
     } else {
-      names.push(source);
+      names.push(source.toLowerCase());
       nodes.push({
-        id: source,
+        id: source.toLowerCase(),
         label: source,
         isTF: true
       });
       nodeId[source] = nodes.length - 1;
     }
     if (!(target in nodeId)) {
-      names.push(target);
+      names.push(target.toLocaleString());
       nodes.push({
-        id: target,
+        id: target.toLowerCase(),
         label: target,
         isTF: false
       });
       nodeId[target] = nodes.length - 1;
     }
     edges.push({
-      id: source + ',' + target,
-      source: source,
-      target: target,
+      id: source.toLowerCase() + ',' + target.toLowerCase(),
```

Comment:
You are repeatedly using source.toLowerCase() and target.toLowerCase(). Why not creating two variables for them?